npm/cypress-schematic/src/builders/builders.json

```diff
@@ -3,7 +3,6 @@
   "builders": {
     "cypress": {
       "implementation": "./cypress",
-      "schema": "./cypress/schema.json",
```

Comment:
any reason we needed to remove this or does this need removed as well? https://github.com/cypress-io/cypress/blob/UNIFY-1291-remove-old-schema/npm/cypress-schematic/src/builders/cypress/schema.json

Reply:
Whoops, added back in. Overzealous "removing schema references", not noticing this was a different schema.